### PR TITLE
Updated node, php and python runtimes

### PR DESCRIPTION
**Pull request by reggeenr**
node -> node:12
python -> python:3.9
php -> php:7.4

### CODE REVIEW

File: runtimes/php/manifest.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Deployment using this manifest file creates following OpenWhisk components:
+4# Deployment using this manifest file creates following OpenWhisk components:
```
Comment (ASEIDEL77) [inline runtimes/php/manifest.yaml]: Typo? I guess the 4 in front of the hash will cause problems...

**Review by ASEIDEL77 — CHANGES_REQUESTED** [review]
See single comment(s)

File: runtimes/php/manifest.yaml (hunk 1 above)
Comment (reggeenr) [reply runtimes/php/manifest.yaml]: good catch. thank you!

**Review by ASEIDEL77 — APPROVED** [review]
lgtm

**Pull request merged**